components/Header/src/index.scss

```diff
@@ -1,13 +1,12 @@
 @import "storybook";
 
 .denhaag-header {
-  background-color: var(--denhaag-header-background-color);
-  border-bottom: 1px solid var(--denhaag-header-border-color);
+  background-color: var(--denhaag-header-background-color, var(--denhaag-color-white));
+  border-bottom: 1px solid var(--denhaag-header-border-color, var(--denhaag-color-grey-1));
   height: auto;
-  padding-block-end: var(--denhaag-header-padding-block);
-  padding-block-start: var(--denhaag-header-padding-block);
-  position: var(--denhaag-header-position);
-  top: var(--denhaag-header-top);
+  padding-block: var(--denhaag-header-padding-block, 0.875rem);
+  position: var(--denhaag-header-position, absolute);
+  top: var(--denhaag-header-top, 0);
```

Comment:
Not bad, maar fallback op 0 is in principe niet nodig ;-)

Reply:
Ik dacht deze is misschien toch handig als ze deze static willen maken of relative via een variabele

Reply:
oh wacht je had het over top :D maar idd.